mava/utils/make_env.py

```diff
@@ -124,6 +131,38 @@ def make_jaxmarl_env(
     return env, eval_env
 
 
+def make_matrax_env(
+    env_name: str, config: DictConfig, add_global_state: bool = False
+) -> Tuple[Environment, Environment]:
+    """
+    Creates Matrax environments for training and evaluation.
+
+    Args:
+        env_name (str): The name of the environment to create.
+        config (Dict): The configuration of the environment.
+        add_global_state (bool): Whether to add the global state to the observation.
+
+    Returns:
+        A tuple of the Matrax environments.
+    """
+    # Select the Matrax wrapper.
+    wrapper = _matrax_registry[env_name]["wrapper"]
+
+    # Create envs.
+    task_name = config["env"]["scenario"]["task_name"]
+    env = matrax.make(task_name, **config.env.kwargs)
+    eval_env = matrax.make(task_name, **config.env.kwargs)
+    env, eval_env = wrapper(env), wrapper(eval_env)
```

Comment:
We can just use the wrapper directly here

```suggestion
    # Create envs.
    task_name = config["env"]["scenario"]["task_name"]
    env = matrax.make(task_name, **config.env.kwargs)
    eval_env = matrax.make(task_name, **config.env.kwargs)
    env, eval_env = MatraxWrapper(env), MatraxWrapper(eval_env)
```